compiler/docgen.nim

```diff
@@ -900,7 +909,11 @@ proc genJsonItem(d: PDoc, n, nameNode: PNode, k: TSymKind): JsonNode =
   result = %{ "name": %name, "type": %($k), "line": %n.info.line.int,
                  "col": %n.info.col}
   if comm.len > 0:
-    result["description"] = %comm
+    try:
+      result["description"] = sanitize(comm)
+    except:
+      echo "Unable to parse: ", getCurrentExceptionMsg()
```

Comment:
when does this happen? in scripts, this might go un-noticed since this succeeds

Reply:
We don't do error reporting via ``echo``.